### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Weirdllamas**
- Fixed the bit about Git Flow
- Fixed the license name and link
- Just some general stuff

### CODE REVIEW

**Comment by Weirdllamas** [comment]
Just a sample basically, i would prefer to work on it more before merge

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -13,20 +13,26 @@ If you are proposing new features, please discuss them with us in the [developme
 ## We Develop with Github
 We use github to host code, to track issues and feature requests, as well as accept pull requests.
 
-## We Use [Github Flow](https://guides.github.com/introduction/flow/index.html), So All Code Changes Happen Through Pull Requests
-Pull requests are the best way to propose changes to the codebase (we use [Github Flow](https://guides.github.com/introduction/flow/index.html)). We actively welcome your pull requests:
+## We Use [Git Flow](https://atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow)
 
-1. Fork the repo and create your branch from `master`.
+![Simple Image Of A Git Flow Workflow](https://wac-cdn.atlassian.com/dam/jcr:61ccc620-5249-4338-be66-94d563f2843c/05%20(2).svg?cdnVersion=532)
```
Comment (Taaku18) [inline CONTRIBUTING.md]: I'd prefer this image actually:

Comment (Weirdllamas) [reply CONTRIBUTING.md]: Should be fixed

**Comment by Weirdllamas** [comment]
Looks good to merge

**Pull request merged**